src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -182,7 +182,7 @@
 
 {% block file_widget -%}
     {%- set attr = attr|merge({class: (attr.class|default('') ~ ' custom-file-input')|trim}) -%}
-    {{- block('form_widget_simple') }}
+    {{ block("form_widget_simple", "form_div_layout.html.twig") }}
```

Comment:
why did you remove the "No leading/trailing whitespace"?
Also double quotes should be single quotes